egs/puebla_nahuatl/st1/local/data_prep.py

```diff
@@ -118,7 +118,7 @@ def ELANProcess(afile, spk_info, spk_details):
             "UtteranceType",
             "Transcripción",
             "Traducción",
-            "Transcripcion",
+            "Transcription",
```

Comment:
This might be true.
@ftshijt, could you check it?

Reply:
Thanks! But this is set with intention as it is in Spanish without the tune. Given it's in the original dataset, I would rather keep the current form.